### PR TITLE
⚡️ Use media network cdn for arweave api

### PR DESCRIPTION
**Pull request by williamchong**
- Reduce anchor api call by sharing anchor for all tx
- Use media network cdn for non-gql calls

### CODE REVIEW

File: src/util/arweave.js [hunk 1]
```diff
@@ -288,32 +301,33 @@ export async function uploadFilesToArweave(files, arweaveIdList, checkDuplicate
...
     let arweaveId = arweaveIds[i];
     let ipfsHash = ipfsHashes[i];
     if (!ipfsHash) {
       ipfsHash = await getFileIPFSHash(f);
     }
     if (!arweaveId) {
       [arweaveId] = await Promise.all([
-        submitToArweave(f, ipfsHash, checkDuplicate),
+        submitToArweave(f, ipfsHash, {
+          checkDuplicate,
```
Comment (WeiJun0827) [inline src/util/arweave.js]: Does `submitToArweave()` accept `{ checkDuplicate }` pareameter?

**Review by WeiJun0827 — APPROVED** [review]

File: src/util/arweave.js [hunk 2]
```diff
@@ -122,7 +129,10 @@ export async function estimateARPrice(data, checkDuplicate = true) {
       };
     }
   }
-  const transaction = await arweave.createTransaction({ data: buffer }, jwk);
+  const transaction = await arweave.createTransaction({
+    data: buffer,
+    last_tx: 'stub_for_estimate',
```
Comment (WeiJun0827) [inline src/util/arweave.js:134]: Curious about the function of adding this config?

Comment (williamchong) [reply src/util/arweave.js:134]: if `last_tx` is not set, arweavejs will query `/tx_anchor` API, which cost ~300ms-3s and has no meaning for estimation

Comment (WeiJun0827) [reply src/util/arweave.js:134]: Got it, thanks!

**Pull request merged**